modules/contact-form/js/gutenblock.jsx

```diff
@@ -0,0 +1,630 @@
+/*global wp*/
+/** @jsx wp.element.createElement */
+/** @format */
+
+/**
+ * External dependencies
+ */
+import {
+	PanelBody,
+	TextControl,
+	TextareaControl,
+	CheckboxControl,
+	ToggleControl,
+	IconButton
+} from '@wordpress/components';
+
+import {
+	registerBlockType,
+	createBlock
+} from '@wordpress/blocks';
+
+import {
+	InnerBlocks,
+	InspectorControls
+} from '@wordpress/editor';
+
+import {
+	Component,
+	Fragment
+} from '@wordpress/element';
+
+import {
+	__
+} from '@wordpress/i18n';
+
+/**
+ * Components:
+ */
+
+class GrunionForm extends Component {
+	constructor( ...args ) {
+		super( ...args );
+		this.onChangeSubject = this.onChangeSubject.bind( this );
+		this.onChangeTo = this.onChangeTo.bind( this );
+		this.onChangeSubmit = this.onChangeSubmit.bind( this );
+	}
+
+	onChangeSubject( x ) {
+		this.props.setAttributes( { subject: x } );
+	}
```

Comment:
We could shorten this a bit like so:
```js
onChangeSubject( subject ) {
    this.props.setAttributes( { subject } );
}
```

What do you think?

Reply:
Sure.  I'm just unfamiliar with that new array syntax.  I'll get it done.

Reply:
💪 you'll love shorthands once you get familiar with them, they're awesome. :-)